### PR TITLE
Add Chinese language.

### PR DESCRIPTION
**Pull request by qiu8310**

### CODE REVIEW

**Comment by nexdrew** [comment]
@qiu8310 Awesome, thanks!

File: README.md [hunk 1]
```diff
@@ -722,6 +722,7 @@ Locales currently supported:
 * **ja:** Japanese.
 * **pt:** Portuguese.
 * **pirate:** American Pirate.
+* **zh:** Chinese.
```
Comment (nexdrew) [inline README.md]: Would you mind putting this _before_ `pirate`? I'd like to give priority to the real locales in the list. :smile:

Comment (qiu8310) [inline README.md]: Updated

Comment (nexdrew) [inline README.md]: Great, thank you!

**Pull request merged**